main/python-bcrypt/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "python-bcrypt"
+pkgver = "4.2.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-setuptools-rust",
+    "python-cffi",
+    "python-build",
+    "python-installer",
+    "cargo",
+]
+makedepends = ["python-devel", "rust-std"]
+depends = ["python-cffi"]
+checkdepends = [*depends, "python-pytest"]
+pkgdesc = "Modern password hashing for software and servers"
+maintainer = "Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license = "Apache-2.0"
+url = "https://github.com/pyca/bcrypt"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "36c757ad7a317627256b7f0be570b16ee1639c6faf398f9f7a7bd4f21bd289ee"
+
+
+def prepare(self):
+    from cbuild.util import cargo
+
+    with self.pushd("src/_bcrypt"):
+        cargo.Cargo(self).vendor()
```

Comment:
```suggestion
    cargo.Cargo(self, wrksrc="src/_bcrypt").vendor()
```

then delete the pushd below in init_build